lib/index.js

```diff
@@ -6,5 +6,6 @@ const mealFetchRequests = require('./fetch-requests/meal-fetch-requests')
 $(document).ready( () => {
   foodFetches.getAllFoods()
   foodListeners.newFoodSubmit()
```

Comment:
possible refactor: wrap all of the event listeners in 1 function, and then call that one function in the index.  Could be called allEventListeners()
But this is really minor